blockexplorer/src/api/index.tsx

```diff
@@ -177,3 +178,29 @@ export const fetchBlocksTransactionsCChain =
     }
     return r;
   };
+
+
+  export async function loadValidatorsInfo() {
+    return new Promise((resolve, reject) => {
+      const urlValidators = `${getBaseUrl()}${baseEndpoint}/validatorsInfo`;
+      let networks = store.getState().appConfig;
+      let activeNetwork = networks.networks.find(
+        element => element.id === networks.activeNetwork,
+      );
+  
+      var request = {
+        method: 'post',
+        url: urlValidators,
+        headers: {
+          'Content-Type': 'application/json',
+        }
+      };
+  
+      axios(request).then(function (response: any) {
+        resolve(response.data.value);
+      }).catch(function (error) {
+        reject([]);
+        console.log(error);
```

Comment:
Please use `console.error` for logging errors

Reply:
Ok, i solved this